templates/Includes/BlogPostPaginatedList.ss

```diff
@@ -3,7 +3,7 @@
         <h2 class="h3 blog-post-heading"><%t BlogPostPaginatedList.POSTS_BY 'Posts by {firstname} {lastname} for {title}' firstname=$CurrentProfile.FirstName lastname=$CurrentProfile.Surname title=$Title %></h2>
     <% end_if %>
     <% loop $PaginatedList %>
-        <article class="blog-entry $EvenOdd">
```

Comment:
You should keep `blog-entry` in case its used somewhere. If you need to modify it you can use a BEM modifier (`--paginated`) rather than underscore.

That means you'll have something that looks like this `blog-entry blog-entry--paginated`.